dashboard/src/actions/fetchPublicDatasets.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import * as types from "./types";
+import API from "../utils/api";
+
+export const fetchPublicDatasets=()=>async(dispatch)=>{
+    try {
+        dispatch({type:types.LOADING});
+        const response=await API.get('api/v1/datasets/list?metadata=dataset.created&access=public',{
```

Comment:
Thanks, @dbutenhof.  How much of that should I be calling for in _this PR_ versus looking for it in a subsequent PR?

For example, should I mark this conversation resolved now?  Should we just ask that @shubham-html-css-js use the JSON object parameter instead of appending to the query string?  Or, should we hold out for the `window.endpoints.api.datasets_list` change?

Reply:
OK, I'm going to deem this resolved, then.

@shubham-html-css-js, if you want to replace the concatenated URL with the `params` object, go for it!